CONTRIBUTING.md

```diff
@@ -69,7 +69,7 @@ that you aren't forgetting to document types and functions. CI will catch it if
 you forget, but the turn around will be a lot slower ;)
 
 ```
-$ cargo build --features "llvm_stable _docs"
+$ cargo build --features docs_
```

Comment:
Yeah the trailing `_` was because leading ones weren't properly handled. Perhaps this should be `testing_only_docs` too?